### PR TITLE
Package requirements

### PR DESCRIPTION
**Pull request by ddathan**
Package requirements for Faculty's submission. Please let us know if there are any issues, thanks!

### CODE REVIEW

File: runtime/environment-cpu.yml [hunk 1]
```diff
@@ -17,5 +17,11 @@ dependencies:
   - scipy=1.9.1
   - tensorflow-cpu=2.10.0
   - tqdm=4.64.1
+  - imbalanced-learn==0.10.0
+  - matplotlib==3.6.2
+  - opacus==1.2.0
+  - sdmetrics==0.8.1
+  - seaborn==0.12.1
```
Comment (jayqi) [inline runtime/environment-cpu.yml:24]: @ddathan Are you generating plots with matplotlib and seaborn as part of your training code? There will not be a way to get any plots out—only text logs.

Comment (ddathan) [reply runtime/environment-cpu.yml:24]: Hi @jayqi, the plots are needed for our development and final report. Of course we will switch this off for the submission code execution and do not expect to create any plots then

Comment (jayqi) [reply runtime/environment-cpu.yml:24]: In order to make the runtime environment as simple as possible to support solutions, it doesn't make sense for us to include dependencies for detailed reporting. I encourage use to run these reporting tools separately on the outputs of your local evaluation. 

Can you clarify if the `sdmetrics` package is also used in this way?

**Comment by jayqi** [comment]
Hi @ddathan. I am closing this PR because I have combined some of your changes with other people's changes in #9, which has been merged.  

Per the previous comment, I have left out matplotlib, seaborn, and sdmetrics. Because this runtime definition needs to be shared by all participants for evaluation, it makes sense to make it minimal for meeting solutions' requirements. Even if you were to create plots during runs in the hosted infrastructure, there would be no way to download them. I encourage your team to either separately run your reporting after any local runs of the evaluation, or to add these dependencies to a local copy of the runtime image that you use on your own. Let me know if you have any additional questions or followup.

**Comment by ddathan** [comment]
Hi @jayqi, no problem, we'll remove the requirement for them in our submission code and just use them locally. Thanks!

**Pull request closed without merging**